src/client/components/app/Transactions/components/TxTTLInput.tsx

```diff
@@ -123,7 +123,6 @@ export interface TxTokenInputProps {
   hideAmount?: boolean;
   loading?: boolean;
   loadingText?: string;
-  ttlType: boolean;
```

Comment:
we shouldnt need a type for ttl. Why cant it just be a number/string?

Reply:
I removed the TTLtype, I had some logic in the component that shouldn't have been there. I believe this has been resolved